inspirehep/modules/disambiguation/config.py

```diff
@@ -25,5 +25,11 @@
 from __future__ import absolute_import, division, print_function
 
 
+DISAMBIGUATION_CLUSTERS_PATH = '/tmp/clusters.json'
+"""The path to the file that will store the curated clusters."""
+
+DISAMBIGUATION_SIGNATURES_PATH = '/tmp/signatures.jl'
+"""The path to the file that will store the curated signatures."""
+
 DISAMBIGUATION_MODEL_PATH = '/tmp/inspire-disambiguation-model.pkl'
```

Comment:
I think we need to save the linkage function here which is supposed to have a .dat extension.

Reply:
Ah, don't worry too much about this guy, I introduced it in https://github.com/inspirehep/inspire-next/commit/6d7a5444419618705dd3edbfc6db61e2eb5282c4 to prove that we are able to load configuration from disk, but the filename will probably change as needed.